pkg/providers/cloudstack/validator.go

```diff
@@ -176,35 +144,15 @@ func (v *Validator) ValidateClusterMachineConfigs(ctx context.Context, cloudStac
 	}
 
 	for _, machineConfig := range cloudStackClusterSpec.machineConfigsLookup {
-		if machineConfig.Namespace != cloudStackClusterSpec.Cluster.Namespace {
-			return fmt.Errorf(
-				"CloudStackMachineConfig %s and Cluster objects must have the same namespace: CloudStackMachineConfig namespace=%s; Cluster namespace=%s",
-				machineConfig.Name,
-				machineConfig.Namespace,
-				cloudStackClusterSpec.Cluster.Namespace,
-			)
-		}
```

Comment:
Removing ip, namespace & machineGroupRef checks because that is handled in the config manager validations for the cluster and cloudstackmachineconfig entries: https://github.com/aws/eks-anywhere/blob/main/pkg/cluster/default_manager.go#L17